carbon-design-system/carbon
# fix: slider overlaying in warn and error icon

Closes #15245 

Slider overlaying in warn and error icon


#### Testing / Reviewing

1. Open Slider with two Handlers
2. Enter a wrong value in minimum input area
3. It should show the invalid value icon
4. Now remove the value from that input
5. It should remove the invalid icon now and should show only error icon.
